src/github/client/github-client.test.ts

```diff
@@ -1,27 +1,86 @@
 /* eslint-disable jest/no-standalone-expect */
-import { getLogger } from "config/logger";
-import { GitHubInstallationClient } from "./github-installation-client";
-import { getInstallationId } from "./installation-id";
 import { HttpsProxyAgent } from "https-proxy-agent";
 import "config/env";
 
+import * as axios from "axios";
+import { GitHubClient, GitHubConfig } from "~/src/github/client/github-client";
+import { booleanFlag } from "config/feature-flags";
+
+jest.mock("axios");
+
 jest.mock("config/feature-flags");
 
+class TestGitHubClient extends GitHubClient {
+	constructor(config: GitHubConfig) {
+		super(config);
+	}
+	public doTestGraphqlCall() {
+		return this.graphql("foo", {});
+	}
+}
+
+const TEST_API_URL = 'http://api.myBaseUrl.com';
+const TEST_GRAPHQL_URL = 'http://graphql.myBaseUrl.com';
+
+const TEST_GITHUB_CONFIG = {
```

Comment:
Should we define this object in setup? Then it's globally accessible and doesn't need to be defined in individual tests.

Reply:
Nah, people will forget anyway and will create their own constants. I don't think we have a single place for such thing, do we?

Reply:
we do, in `test/setup.ts`.  It's where we set globals for tests like `jiraHost`.  I could see this being useful to have a `githubConfig` and `gheConfig` object premade to be easily used in tests :)

Reply:
Gotcha, will change then. Thanks, @mboudreau !